### PR TITLE
RSDEV-369, RSDEV-370: Integrate FIeldmark app and create app banner (disabled for now)

### PR DESCRIPTION
**Pull request by rs-nicof**

### CODE REVIEW

**Comment by rlamacraft** [comment]
Some of the commits haven't made it into this PR, did you force push without fetching first?

**Comment by rs-nicof** [comment]
✅   Successfully Tested on AWS https://rsdev-369-integrate-fieldmark-uncommented-2.researchspace.com/ ✅ 

### Test strategy
 - Uncommented the following line on `settings_mod.js` 
 - - image
 - Verify there is the option `fieldmark.available` in the configuration section 
 - Verify the Fieldmarkbanner works as expected in the `Apps` page
 
 ### Test evidence

https://github.com/user-attachments/assets/4debae1a-4b98-47ef-8be1-ce18494c586b

**Review by mKowalski256 — APPROVED** [review]
Looks good to me, one comment about naming of a deployment property.

File: src/main/webapp/ui/package-lock.json [hunk 1]
```diff
@@ -3505,7 +3505,7 @@
       "version": "1.0.11",
       "resolved": "https://registry.npmjs.org/@mapbox/node-pre-gyp/-/node-pre-gyp-1.0.11.tgz",
       "integrity": "sha512-Yhlar6v9WQgUp/He7BdgzOz8lqMQ8sU+jkCq7Wx8Myc5YFJLbEe7lgui/V7G1qB1DJykHSGwreceSaD60Y0PUQ==",
-      "devOptional": true,
+      "optional": true,
```
Comment (rs-fraser) [inline src/main/webapp/ui/package-lock.json:3508]: Just want to highlight all of these changes `devOptional` -> `optional` as it looks like the kind of thing that could sneak in unintentionally

**Review by rs-fraser — APPROVED** [review]
Looks good. Tested locally and can enable the property and see the app page placeholder. 

Left 1 comment just checking the changes in `package-lock.json` are intentional.

File: src/main/webapp/ui/package-lock.json (hunk 1 above)
Comment (rs-nicof) [reply src/main/webapp/ui/package-lock.json:3508]: @rlamacraft can you confirm that change is due?

Comment (rlamacraft) [reply src/main/webapp/ui/package-lock.json]: I've re-run `npm install --force`, whichever changes persist are required

**Pull request merged**